pages/affiliation-link.vue

```diff
@@ -198,6 +230,23 @@ const productId = computed(() => {
   }
   return input
 })
+
+const linkQueryInput = ref('')
```

Comment:
Should we add a default utm_medium? `qrcode` or sth

Reply:
Good suggestion